src/software/proto/message_translation/ssl_detection_message_translator.cpp

```diff
@@ -0,0 +1,87 @@
+#include "software/proto/message_translation/ssl_detection_message_translator.h"
+
+#include "shared/constants.h"
+
+std::unique_ptr<SSL_DetectionBall> createDetectionBall(const BallState& ball)
+{
+    auto detection_ball = std::make_unique<SSL_DetectionBall>();
+
+    // Give all detections "perfect" confidence since we have no means of reasonably
+    // esimating confidence
+    detection_ball->set_confidence(1.0);
+    // We don't have the information to simulate a camera and therefore the area of the
+    // ball
+    detection_ball->clear_area();
+    auto x_position_mm = static_cast<float>(ball.position().x() * MILLIMETERS_PER_METER);
+    auto y_position_mm = static_cast<float>(ball.position().y() * MILLIMETERS_PER_METER);
+    detection_ball->set_x(x_position_mm);
+    detection_ball->set_y(y_position_mm);
+    detection_ball->set_z(static_cast<float>(ball.height() * MILLIMETERS_PER_METER));
+    // grSim uses the ball's position as its pixel values, so we do the same thing here
```

Comment:
uber nit: I think we should remove this reference to grSim, the simulator will be replacing it entirely and future members won't need to bother looking up what grSim is.

Reply:
done